downstream/assemblies/platform/assembly-controller-organizations.adoc

```diff
@@ -8,15 +8,15 @@ ifdef::context[:parent-context: {context}]
 
 :context: access-mgmt-orgs
 
-An organization is a logical collection of users, teams, and resources. It is the highest level object in the {PlatformNameShort} object hierarchy. After you have created an organization, {PlatformNameShort} displays the organization details. You can then manage access and execution environments for the organization.
-{PlatformNameShort} automatically creates a default organization and the system administrator is automatically assigned to this organization. If you have a Self-support level license, you have only the default organization available and must not delete it.
-
-// [ddacosta] Removed this statement because I think it was relevant when this content was upstream but in the downstream docs, it’s implied that you have a license.
-//[NOTE]
-//====
-//Only Enterprise or Premium licenses can add new organizations.
-//====
-//Enterprise and Premium license users who want to add a new organization should refer to the xref:proc-controller-create-organization[Creating an organization].
+An organization is a grouping of resources such as projects, inventories, and credentials. You can define granular access to the resources wihtin an organization by first assigning users and teams to the organization, and then assigning those users and teams defined roles within the organization.
+
+As an administrator you can use organizations to organize groups of resources. 
+Assigning a team or user to the organization gives the team or user access to the resources within the organization.
+This way, administrators no longer have to grant individual teams and users access to new resources as they become available; teams and users gain access to new resources when they are added to the organization.
+
+After you have created an organization, {PlatformNameShort} displays the organization details. You can then manage resources such as access and execution environments for the organization.
+
+{PlatformNameShort} automatically creates a default organization. If you have a Self-support level license, you have only the default organization available and must not delete it.
```

Comment:
Any specific reason for capitalizing 'Self'? I've seen it capitalized differently here, so you might wanna keep it consistent across: https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.5/html-single/access_management_and_authentication/index#assembly-controller-teams_gw-manage-rbac.